src/v/kafka/server/handlers/fetch.h

```diff
@@ -219,6 +219,11 @@ struct read_result {
     using foreign_data_t = ss::foreign_ptr<std::unique_ptr<iobuf>>;
     using data_t = std::unique_ptr<iobuf>;
     using variant_t = std::variant<data_t, foreign_data_t>;
+    struct memory_units_t {
+        ssx::semaphore_units kafka;
+        ssx::semaphore_units fetch;
```

Comment:
what's the "kafka" units? is that total semaphore?

Reply:
yes, the total one in `net::server` for kafka